src/Avalonia.FuncUI/DSL/TextBox.fs

```diff
@@ -68,12 +69,75 @@ module TextBox =
+
+        static member innerLeftContent<'t when 't :> TextBox>(value: obj) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<obj>(TextBox.InnerLeftContentProperty, value, ValueNone)
+
+        static member innerLeftContent<'t when 't :> TextBox>(view: IView option) : IAttr<'t> =
+            AttrBuilder<'t>.CreateContentSingle(TextBox.InnerLeftContentProperty, view)
+
+        static member innerLeftContent<'t when 't :> TextBox>(view: IView) : IAttr<'t> =
+            Some view |> TextBox.innerLeftContent
+
+        static member innerRightContent<'t when 't :> TextBox>(text: string) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<string>(TextBox.InnerRightContentProperty, text, ValueNone)
+
+        static member innerRightContent<'t when 't :> TextBox>(value: obj) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<obj>(TextBox.InnerRightContentProperty, value, ValueNone)
+
+        static member innerRightContent<'t when 't :> TextBox>(view: IView option) : IAttr<'t> =
+            AttrBuilder<'t>.CreateContentSingle(TextBox.InnerRightContentProperty, view)
+
+        static member innerRightContent<'t when 't :> TextBox>(view: IView) : IAttr<'t> =
+            Some view |> TextBox.innerRightContent
+
+        static member revealPassword<'t when 't :> TextBox>(value: bool) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<bool>(TextBox.RevealPasswordProperty, value, ValueNone)
+
+        static member isUndoEnabled<'t when 't :> TextBox>(value: bool) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<bool>(TextBox.IsUndoEnabledProperty, value, ValueNone)
+
+        static member undoLimit<'t when 't :> TextBox>(value: int) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<int>(TextBox.UndoLimitProperty, value, ValueNone)
+        
+        static member onCopyingToClipboard<'t when 't :> TextBox>(func: RoutedEventArgs -> unit, ?subPatchOptions) =
+            AttrBuilder<'t>.CreateSubscription<RoutedEventArgs>(TextBox.CopyingToClipboardEvent, func, ?subPatchOptions = subPatchOptions)
+
+        static member onCuttingToClipboard<'t when 't :> TextBox>(func: RoutedEventArgs -> unit, ?subPatchOptions) =
+            AttrBuilder<'t>.CreateSubscription<RoutedEventArgs>(TextBox.CuttingToClipboardEvent, func, ?subPatchOptions = subPatchOptions)
+
+        static member onPastingFromClipboard<'t when 't :> TextBox>(func: RoutedEventArgs -> unit, ?subPatchOptions) =
+            AttrBuilder<'t>.CreateSubscription<RoutedEventArgs>(TextBox.PastingFromClipboardEvent, func, ?subPatchOptions = subPatchOptions)
+
         static member onTextChanged<'t when 't :> TextBox>(func: string -> unit, ?subPatchOptions) =
             AttrBuilder<'t>.CreateSubscription<string>(TextBox.TextProperty, func, ?subPatchOptions = subPatchOptions)
 
+        static member onTextChanged<'t when 't :> TextBox>(func: TextChangedEventArgs -> unit, ?subPatchOptions) =
```

Comment:
The second version of onTextChanged seems to have caused some confusion in some of the sample apps, e.g.

![image](https://github.com/fsprojects/Avalonia.FuncUI/assets/1178570/1a4f3c7b-a018-4b8c-afbc-193888757ebd)

Reply:
I would like to exclude the changes to onTextChanged from this PR due to its wide impact. I might submit a separate PR for it later.